### PR TITLE
Fkm 21

### PR DESCRIPTION
**Pull request by AndrewFall**

### CODE REVIEW

File: drupal/modules/taxonomy/taxonomy.module.orig [hunk 1]
```diff
@@ -0,0 +1,2059 @@
+<?php
```
Comment (temoor-gilmutdinov) [inline drupal/modules/taxonomy/taxonomy.module.orig]: .orig files are unnecessary.

File: drupal/profiles/favrskov/modules/contrib/apachesolr_attachments/solrconfig.tika.patch [hunk 2]
```diff
@@ -1,22 +0,0 @@
-diff --git a/solrconfig.xml b/solrconfig.xml
```
Comment (temoor-gilmutdinov) [inline drupal/profiles/favrskov/modules/contrib/apachesolr_attachments/solrconfig.tika.patch]: This patch is purposed for ApacheSolr core configs, keep it.

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/87/

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/88/

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/89/

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/90/

File: drupal/profiles/favrskov/patches/media_browser_plus/2387943-subfolder-search-12.patch [hunk 3]
```diff
@@ -10,7 +10,7 @@ index 008794a..7ef7e8f 100644
 +    $folders = array($this->value);
 +    if (isset($this->view->exposed_data['mbp_current_folder_include_subfolders']) && $this->view->exposed_data['mbp_current_folder_include_subfolders'] !== 0) {
 +      $taxonomy_tree = taxonomy_get_tree(taxonomy_vocabulary_machine_name_load('media_folders')->vid, $this->value);
-+      if (is_array($taxonomy_tree)) {
++      if (is_agit rray($taxonomy_tree)) {
```
Comment (temoor-gilmutdinov) [inline drupal/profiles/favrskov/patches/media_browser_plus/2387943-subfolder-search-12.patch]: What's that?

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/91/

**Pull request merged**